src/machine/machinectl.c

```diff
@@ -140,6 +192,8 @@ static int list_machines(int argc, char *argv[], void *userdata) {
                                &error,
                                &reply,
                                NULL);
+
+
```

Comment:
spurious newline...

Note that we generally try to keep the function called and the respective error checking together, hence there really shouldn't be any empty line at this very spot, and certainly not 2...